java/src/com/ibm/streamsx/topology/TStream.java

```diff
@@ -507,6 +507,32 @@
      *             than 1.0.
      */
     TStream<T> sample(double fraction);
+    
+    /**
+     * Return a TStream that runs in a separate process from the parent. The
```

Comment:
parent/child hasn't been used to describe stream relationships, so it's not clear what defines a parent-child relationship. I'm not sure we need to define such a relationship either.

I think all that isolate() defines is that any processing applied to the returned stream will be in a separate process to the processing that produced this stream.

Where there are no specific instructions from the application, the Java Application API is free to decide the layout of a topology to PEs, thus it's not guaranteed that a stream's "children" will be in the the same PE.